src/main/java/org/opensearch/searchrelevance/judgments/clickmodel/coec/CoecClickModelParameters.java

```diff
@@ -25,6 +28,19 @@ public CoecClickModelParameters(final int maxRank) {
         this.maxRank = maxRank;
     }
 
+    /**
+     * Creates new parameters.
+     * @param maxRank The max rank to use when calculating the judgments.
+     * @param startDate The start date for filtered date range.
+     * @param endDate The end date for filtered date range.
+     * of UBI event dates to consider.
```

Comment:
this line looks like it's kind of hanging?  Can you edit lines 32 and 36 into one line in 32 that describes what this does?